paddle/fluid/framework/downpour_worker.cc

```diff
@@ -72,6 +74,85 @@ void DownpourWorker::Initialize(const TrainerDesc& desc) {
   scale_datanorm_ = desc.scale_datanorm();
   dump_slot_ = desc.dump_slot();
   adjust_ins_weight_config_ = desc.adjust_ins_weight_config();
+  dump_fields_path_ = desc.dump_fields_path();
+  dump_fields_.resize(desc.dump_fields_size());
+  for (int i = 0; i < desc.dump_fields_size(); ++i) {
+    dump_fields_[i] = desc.dump_fields(i);
+  }
+  need_dump_field_ = false;
+  if (dump_fields_.size() != 0 && dump_fields_path_ != "") {
+    need_dump_field_ = true;
+  }
+  mpi_rank_ = desc.mpi_rank() / 2;
+}
+
+template <typename T>
+std::string print_lod_tensor_type(LoDTensor* tensor, int64_t start,
```

Comment:
code style of function name is wrong,  so as other functions

Reply:
done